include/aomusicplayer.h

```diff
@@ -19,13 +19,20 @@ class AOMusicPlayer {
   AOMusicPlayer(QWidget *parent, AOApplication *p_ao_app);
   virtual ~AOMusicPlayer();
   void set_volume(int p_value, int channel = -1);
-  void set_looping(bool toggle, int channel = 0);
+  void set_looping(bool loop_song, int channel = 0);
   void set_muted(bool toggle);
 
   const int m_channelmax = 4;
 
   // These have to be public for the stupid sync thing
+  /**
+   * @brief The starting sample of the AB-Loop.
+   */
   int loop_start[4] = {0, 0, 0, 0};
+
+  /**
+   * @brief The end sample of the AB-Loop.
+   */
   int loop_end[4] = {0, 0, 0, 0};
```

Comment:
will this ever go negative or to -1?
if not please make it unsigned

Reply:
If people are stupid enough to put it in the txt file for it, sure. Afaik this purely loads from the AB sample configuration, so this should never, in any case, be negative as it would completely fail the idea of the AB loop.

Reply:
making it unsigned also allows the value to be double as high and prevents funky overflows